.github/scripts/auto-backport.py

```diff
@@ -49,7 +49,7 @@ def create_pull_request(repo, new_branch_name, base_branch_name, pr, backport_pr
         backport_pr.add_to_assignees(pr.user)
         if is_draft:
             backport_pr.add_to_labels("conflicts")
-            pr_comment = f"@{pr.user} - This PR was marked as draft because it has conflicts\n"
+            pr_comment = f"@{pr.user.login} - This PR was marked as draft because it has conflicts\n"
```

Comment:
i am pasting related schema here
```yaml
    "user": {
      "title": "Simple User",
      "description": "A GitHub user.",
      "type": "object",
      "properties": {
        "name": {
          "type": [
            "string",
            "null"
          ]
        },
        "email": {
          "type": [
            "string",
            "null"
          ]
        },
        "login": {
          "type": "string",
          "examples": [
            "octocat"
          ]
        },
        "id": {
          "type": "integer",
          "format": "int64",
          "examples": [
            1
          ]
        },
```

see also https://pygithub.readthedocs.io/en/stable/github_objects/PullRequest.html#pullrequest and https://docs.github.com/en/rest/pulls/pulls?apiVersion=2022-11-28#get-a-pull-request